Web/Edubase.Web.UI/Assets/Scripts/Entry/create-establishment.js

```diff
@@ -1,3 +1,7 @@
+import Vue from 'vue';
+const ceApp = new Vue({
+  el: '#create-establishment'
+});
```

Comment:
we should removed this actually as we are no longer using Vue cloak to show/hide the JS version